blocks/app-download/app-download.js

```diff
@@ -0,0 +1,25 @@
+export default function decorate(block) {
+  const qrImage = block.querySelector('picture');
+  qrImage.parentNode.className = 'qrcode';
+  const firstButtonContainer = block.querySelector('p.button-container');
+  // Get its parent div
+  const storeDiv = firstButtonContainer.parentNode;
+  storeDiv.classList.add('store');
+  // Select all button container paragraphs
+  const buttonContainers = storeDiv.querySelectorAll('p.button-container');
+  // Iterate through each button container
+  buttonContainers.forEach((buttonContainer) => {
+    // Get the anchor tag inside the button container
+    const anchorTag = buttonContainer.querySelector('a');
+    const picture = document.createElement('picture');
+    const img = document.createElement('img');
+    img.src = `/icons/${anchorTag.text}.png`;
```

Comment:
@ravverma can we use decorateIcons method here ?

Reply:
these are images in the original side, so either put in git or on doc.
but it will not change that frequently so that put in git.
I can try to get them from doc.

Reply:
@anagarwa  these are the badge provided by https://play.google.com/intl/en_us/badges/ which comes in png/eps format only.
PNG file can't in doc.

Reply:
Used `createOptimizedImage` instead @anagarwa